client/blocks/inline-help/popover.jsx

```diff
@@ -132,6 +153,15 @@ class InlineHelpPopover extends Component {
 					setStoredTask={ setStoredTask }
 				/>
 
+				{ 'gutenberg' === this.props.selectedEditor && (
+					<Button
+						onClick={ this.switchToClassicEditor }
+						className="inline-help__classic-editor-toggle"
+					>
+						{ translate( 'Switch to the Classic Editor' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **2** times:
`translate( 'Switch to Classic Editor' )` *ES Score: 15*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
+1 to this.
We're using "Switch to Classic Editor" in the Calypsoify Gutenberg's More menu:

![image](https://user-images.githubusercontent.com/2070010/48346298-91b4b700-e672-11e8-94f9-fd49bed1244e.png)

Reply:
Per our quick discussion in Slack, I'm not sure if there's an actionable item here. I imagine you would still want to translate a string, even if that same string has been translated elsewhere in the application. I'll look into that more though!

Reply:
A8C i18n bot says that there is currently a message very similar that is already translated, so it's suggesting to use that message: 
```suggestion
						{ translate( 'Switch to Classic Editor' ) }
```

That would prevent to create a new message that would be needed to translate.

Reply:
Ah, thanks for this clarification @mmtr - I was totally overlooking the "the" I had in the string and through it was the same, which is what threw me off!

Just pushed an update to use the previously translated string.